components/AppBar/README.md

```diff
@@ -1,12 +1,13 @@
 <!--docs:
-title: "App Bar"
+title: "App Bars"
```

Comment:
I went with convention of the guidelines, which uses plural URLs and headings in most places. Do you think that's reasonable?

<img width="274" alt="screen shot 2017-04-24 at 10 57 08 am" src="https://cloud.githubusercontent.com/assets/42326/25351209/d20405ea-28dc-11e7-9fa0-41260ba3b56c.png">